packages/charts/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -751,32 +748,27 @@ export class VerticalStackedBarChartBase
     this._lineHoverFocus(lineData, mouseEvent);
   };
 
-  private _lineHoverOut = () => {
```

Comment:
is setting of these state variables not needed anymore. _handleMouseOut is not resetting this values.

Reply:
_lineHoverOut caused the callout to flicker because the elements are small. So, I used _handleMouseOut to close the callout only when the mouse leaves the chart area.